src/data/github-projects.json

```diff
@@ -544,6 +544,31 @@
             ],
             "repoName": "docToolchain/aoc-2019",
             "logo": "advent-of-code.png"
+        },
+        {
+            "title": "gosh",
+            "description": "Do everything from the terminal",
```

Comment:
As a new contributor, I think I'd be a bit confused by this description. (E.g. "This projects seems to want me to do more things from my terminal. But what is this project even about? Is it a sort of developer-life guidebook?")

Maybe the more explicit description from https://www.gitpod.io/docs/languages/go/#example-repositories could be helpful here too?

Reply:
The end goal will be enabling users to switch from GUI to CLI it will have facilities for playing music, doing math, web browsing, etc.